enhancements/installer/image-based-installer.md

```diff
@@ -0,0 +1,352 @@
+OpenShift from the [seed OCI image](https://github.com/openshift-kni/lifecycle-agent/blob/main/docs/seed-image-generation.md)
+and optionally precache all release container images under the
+`/var/lib/containers` directory.
+
+The installation ISO approach is very similar to what is already implemented by
+the functionality of the [Agent-based Installer](/enhancements/agent-installer-agent-based-installer.md), although
+IBI differs from the OpenShift Agent-based Installer in several key aspects:
+
+- while the Agent-based Installer may offer flexibility and versatility in certain scenarios,
+  it may not meet the stringent time constraints and requirements of far-edge deployments
+  in the telecommunications industry due to the inherently long installation process,
+  exacerbated by low bandwidth and high packet latency.
+- with the Agent-based Installer all cluster configuration needs to be provided upfront
+  during the generation of the ISO image, while with IBI the cluster
+  configuration is provided in an additional step.
+
+IBI offers key advantages, where fast and reliable deployment at the edge is
+crucial. By generating ISO images containing all the necessary components,
+IBI significantly accelerates deployment times. Moreover, unlike the Agent-based
+Installer, the image-based approach allows for cluster configuration to be
+supplied upon deployment at the edge, rather than during the initial ISO
+generation process. This flexibility enables operators to use a single generic
+image for installing multiple clusters, streamlining the deployment process and
+reducing the need for multiple customized ISO images.
+
+The command-line tool will also support generating a configuration ISO with all
+the site specific configuration data for the cluster to be installed provided as
+input. The configuration ISO contents are the following:
+- `ClusterInfo` (cluster name, base domain, hostname, node IP)
+- SSH `authorized_keys`
+- Pull Secret
+- Extra Manifests
+- Generated keys and certs (compatible with the generated admin `kubeconfig`)
+- Static networking configuration
+
+The site specific configuration data will be generated according to information
+provided in the `install-config.yaml` and the manifests provided in the
+installation directory as input. To complete the installation at the edge site:
+- the cluster configuration for the edge location can be delivered by copying
+  the config ISO content onto the node and placing it under `/opt/openshift/cluster-configuration/`.
+- the cluster configuration can also be delivered using an attached ISO, a
+  systemd service running on the host pre-installed and IBI will
+  mount that ISO (identified by a known label) and copy the cluster configuration
+  to `/opt/openshift/cluster-configuration/`.
+- the cluster configuration data on the disk will be used to configure the
+  cluster and allow OCP to start successfully.
+
+### Workflow Description
```

Comment:
in my opinion, the Workflow Description section is typically the most helpful part of an enhancement. It's not necessary to have every detail specified, but I think it would be good to list out the high-level steps that would be involved with installing a cluster in this manner.

Reply:
Absolutely, I have added a high-level workflow description for this installation method. Please let me know if this is too high-level or if you think it's missing anything.